### PR TITLE
OCD-3430: Remove ERD and ERD+1week Cures-specific jobs

### PR DESCRIPTION
**Pull request by kekey1**

### CODE REVIEW

File: chpl/chpl-resources/src/main/resources-production/log4j2.xml [hunk 1]
```diff
@@ -483,34 +406,10 @@
...
-        </Logger>
-        <Logger name="createTestToolJobLogger" level="INFO"
-            additivity="false">
-            <AppenderRef ref="createTestToolJob" />
-        </Logger>
-        <Logger name="removeCriteriaJobLogger" level="INFO"
-            additivity="false">
-            <AppenderRef ref="removeCriteriaJob" />
-        </Logger>
         <Logger name="modifyTestFunctionalityJobLogger" level="INFO"
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources-production/log4j2.xml]: Does this job still exist?

File: chpl/chpl-resources/src/main/resources-qa/log4j2.xml [hunk 2]
```diff
@@ -523,9 +422,5 @@
             additivity="false">
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources-qa/log4j2.xml:392]: DO these three jobs exist?

File: chpl/chpl-resources/src/main/resources-production/log4j2.xml (hunk 1 above)
Comment (kekey1) [reply chpl/chpl-resources/src/main/resources-production/log4j2.xml]: It gets removed in a different ticket OCD-3425

File: chpl/chpl-resources/src/main/resources-qa/log4j2.xml (hunk 2 above)
Comment (kekey1) [reply chpl/chpl-resources/src/main/resources-qa/log4j2.xml:392]: They all get removed in a different ticket

**Review by andlar — DISMISSED** [review]

**Review by tmy1313 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by drbgfc — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by drbgfc — APPROVED** [review]

**Pull request merged**